src/ints/int10_memory.cpp

```diff
@@ -22,112 +22,179 @@
 #include "mem.h"
 #include "inout.h"
 
-static uint8_t static_functionality[0x10]=
-{
- /* 0 */ 0xff,  // All modes supported #1
- /* 1 */ 0xff,  // All modes supported #2
- /* 2 */ 0x0f,  // All modes supported #3
- /* 3 */ 0x00, 0x00, 0x00, 0x00,  // reserved
- /* 7 */ 0x07,  // 200, 350, 400 scan lines
- /* 8 */ 0x04,  // total number of character blocks available in text modes
- /* 9 */ 0x02,  // maximum number of active character blocks in text modes
- /* a */ 0xff,  // Misc Flags Everthing supported 
- /* b */ 0x0e,  // Support for Display combination, intensity/blinking and video state saving/restoring
- /* c */ 0x00,  // reserved
- /* d */ 0x00,  // reserved
- /* e */ 0x00,  // Change to add new functions
- /* f */ 0x00   // reserved
+// clang-format off
+static uint8_t static_functionality[0x10] = {
```

Comment:
```suggestion
static const std::array<uint8_t, 0x10> static_functionality = {
```

Reply:
Doing the `static const` part, but leaving it as a vector. I dislike having to specify the size; that's error-prone. `std::vector` is just an array if you never resize it, so personally I'd just default to `std::vector` everywhere.